### PR TITLE
update docs, describe how to use horizontalAlign/verticalAlign

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Update docs to describe the latest features introduced for positioning & overlapping (#38)

### CODE REVIEW

File: iron-fit-behavior.html [hunk 1]
```diff
@@ -63,23 +80,25 @@
...
        */
       positionTarget: {
-        type: Node
+        type: Element
       },
 
       /**
        * The orientation against which to align the element horizontally
-       * relative to the `positionTarget`. Possible values are left, right, auto.
+       * relative to the `positionTarget`. Possible values are `left, right, auto`.
```
Comment (cdata) [inline iron-fit-behavior.html]: Nit pick: technically the possible values should be expressed as string literals in this doc.

File: iron-fit-behavior.html [hunk 2]
```diff
@@ -225,7 +244,7 @@
     },
 
     /**
-     * Fits and optionally centers the element into the window, or `fitInfo` if specified.
+     * Positions and fits the element into `fitInfo`.
```
Comment (cdata) [inline iron-fit-behavior.html]: `fitInfo` should be `fitInto` here, I think?

**Comment by cdata** [comment]
Small tweaks requested, otherwise LGTM 👍

File: iron-fit-behavior.html (hunk 2 above)
Comment (valdrinkoshi) [inline iron-fit-behavior.html]: Wow was there since ages, didn't notice it! good catch ;)

**Pull request merged**